src/inference/src/dev/threading/thread_affinity.cpp

```diff
@@ -121,7 +121,7 @@ std::tuple<CpuSet, int> get_process_mask() {
 }
 void release_process_mask(cpu_set_t* mask) {
     if (nullptr != mask)
-        CPU_FREE(mask);
+        delete mask;
```

Comment:
manual new / delete are unsafe, is it possible to wrap to unique_ptr / shared_ptr ?

Reply:
release_process_mask is the delete function of [using CpuSet = std::unique_ptr<cpu_set_t, ReleaseProcessMaskDeleter>;](https://github.com/openvinotoolkit/openvino/blob/8d0381b0fee3ce4e68bc5705f293c2eb3bcea9d3/src/inference/src/dev/threading/thread_affinity.hpp#L56)

Reply:
Ok, I removed the manual delete. Please have a look.

Reply:
how about change `new DWORD_PTR(pro_mask)` to std::make_unique https://www.enseignement.polytechnique.fr/informatique/INF478/docs/Cpp/en/cpp/memory/unique_ptr/make_unique.html ?

Reply:
Ok, updated, thank you!